lib/v2/screens/dashboard/interactor/viewmodels/wallet_event.dart

```diff
@@ -9,8 +9,8 @@ abstract class WalletEvent extends Equatable {
   List<Object> get props => [];
 }
 
-class RefreshDataEvent extends WalletEvent {
-  const RefreshDataEvent();
+class OnLoadWalletData extends WalletEvent {
+  const OnLoadWalletData();
   @override
-  String toString() => 'RefreshDataEvent';
-}
\ No newline at end of file
+  String toString() => 'OnLoadWalletData';
+}
```

Comment:
@n13 I change the name of this event.
We should not use such generic names for events, then it is confusing in the logs and I don't really know what screen or component is launching such an event.

![image](https://user-images.githubusercontent.com/42857405/124653305-c351a080-de62-11eb-8ba4-64bfc973f4a6.png)